packages/pluggableWidgets/datagrid-web/src/Datagrid.tsx

```diff
@@ -56,7 +56,11 @@ export default function Datagrid(props: DatagridContainerProps): ReactElement {
                         ) : (
                             <span className="td-text">{column.attribute(value).displayValue}</span>
                         ),
-                        classNames(props.rowClass?.(value)?.value, column.columnClass?.(value)?.value)
+                        classNames(
+                            column.alignment ? `align-column-${column.alignment}` : "",
```

Comment:
I think the ternary assignment is not necessary here. `column.alignment` will always have one of these values: ` "left" | "center" | "right"`

Reply:
You're right :)